flow/project.py

```diff
@@ -1410,7 +1391,21 @@ def _add_dummy_operation(job):
         if n > 0:
             context['op_counter'].append(('[{} more operations omitted]'.format(n), ''))
 
-        print(template.render(**context), file=file)
+        status_output = template.render(**context)
+        print(status_output, file=file)
```

Comment:
here we have 3 consecutive prints to the specified file. Is this for comparison? There is no logic to only have one print which is the final idea correct?

Reply:
Yes these are currently my tests on different packages and different output syntax(markdown, html or terminal rendering). I will add logic to one of them based on user input later once we decide the best way for all of these output rendering.